src/all/pornpics/src/eu/kanade/tachiyomi/extension/all/pornpics/JsonFileLoader.kt

```
@@ -0,0 +1,20 @@
+package eu.kanade.tachiyomi.extension.all.pornpics
+
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromStream
+import uy.kohesive.injekt.injectLazy
+
+object JsonFileLoader {
+    val json: Json by injectLazy()
+    val classLoader = this::class.java.classLoader!!
+
+    inline fun <reified T> loadJsonAs(fileName: String): T {
+        val fileContent = classLoader.getResourceAsStream(fileName)
+        return json.decodeFromStream<T>(fileContent)
```

How does this handle errors? Failing loudly might be valid.